test/cpu_debugger.cpp

```diff
@@ -68,6 +79,24 @@ TEST(debugger, add_breakpoint)
 
     dbg.add_breakpoint(neg);
     dbg.call({result}, {a, b});
+
+    if (is_codegen_mode())
+    {
+        try
+        {
+            ASSERT_EQ(*static_cast<int*>(dbg.inspect(add)), -777);
+        }
+        catch (const std::exception& e)
+        {
+            EXPECT_EQ(e.what(), std::string("_Map_base::at"));
+            return;
+        }
+        catch (...)
+        {
+            FAIL() << "Expected std::exception";
+        }
+    }
```

Comment:
What was the conclusion about unsupported tests? Make them pass?
It we cannot do any better for now, I would suggest that we just return.  Code in line 87/91 seems very fragile to me. If the way of "crashing" changes, we would have to update the test to the new way of crashing :). What do you think?

Reply:
If agreed, I would move this to the beginning of the test.

Reply:
Yes, you can move it. Please put a TODO there to switch to skip once there is a new release of gtest.

Reply:
Sorry, my bad. In my previous message, I wanted to suggest that you could move the code as part of this PR. I didn't mean I wanted to intrude on your changes and do the coding myself, sorry :). Since you are modifying this code, could you please take care of that?